lms/templates/cme_register.html

```diff
@@ -354,7 +347,7 @@ <h3 class="message-title">Welcome ${extauth_id}</h3>
          
           <li class="field select" id="specialty-container">
             <label for="specialty">Specialty</label>
-            <select id="specialty" name="specialty" class="controller select-controller">
```

Comment:
before we remove the `.controller` classes we should make sure there isn't any CSS that depends on them...